### DISCUSSION TITLE
Help with updating from 5.0.2.2

### DISCUSSION THREAD

**Discussion by lennertfabriek** [bootscore/bootscore#615] [Q&A]
The website villamila.nl is never updated.. now client wants it to be updated. But I tried everything but can't update it without crashing the website. I also tried it step by step with every version. Can you help me with this or (paid) update it for me?

**Comment by crftwrk** [comment]
Yes, this version is more than 2 years old and many things happen since then. 

Firstly, I recommend to install a backup somewhere and do updates on the backup site instead updating the live site. Breaking changes where done in v5.2.0 which you can [download here](https://github.com/bootscore/bootscore/releases/download/v5.2.0.0/bootscore-main.zip).

After install v5.2, do ***ALL*** update steps as [described here](https://bootscore.me/blog/bootstrap/bootscore-5-2-0-0/).

When done and your theme is working, you can safely update to [latest v5.3.x](https://github.com/bootscore/bootscore/releases).

I do not know how deep the modifications in your child are, but why not simply install a fresh child and copy modifications step-by-step  from your existing child to the new one? This seems to be the fastest way. However, you must update WooCommerce to latest version as well.

**Comment by lennertfabriek** [comment]
Thanks! I tried the first option earlier without victory. Last one seems the best. Will try!
